src/02-common-principles.md

```diff
@@ -542,14 +542,25 @@ Describing dates and timestamps:
 -   Date time information MUST be expressed in the following format
     `YYYY-MM-DDThh:mm:ss[.000000]` (year, month, day, hour (24h), minute,
     second, and optionally fractional second).
-    This is equivalent to the RFC3339 "date-time" format, time zone is always
-    assumed as local time).
+    This is almost equivalent to the [RFC3339](https://tools.ietf.org/html/rfc3339)
+    "date-time" format, with the exception that in BIDS, time zone is always
+    assumed as local time.
+    The RFC3339 `Z` syntax to indicate UTC or the RFC3339 "offset" syntax are not
+    supported.
```

Comment:
This feels like it gives a bit too much prominence to this unsupported feature of RFC3339.

```suggestion
    This is almost equivalent to the [RFC3339](https://tools.ietf.org/html/rfc3339)
    "date-time" format, with the exception that UTC offsets are not indicated,
    and the time zone is always assumed to be the local time of the dataset viewer.
```

As I think about this, though, the value of these data are the relative offsets between acquisitions. If an offset change like daylight saving time comes between two sessions, then a user reading the dataset may calculate a different relative elapse of +/- 1 hour, depending on their own local rules.

Would it make sense to change the pattern and text to:

```Markdown
-   Date time information MUST be expressed in the following format
    `YYYY-MM-DDThh:mm:ss[.000000][Z]` (year, month, day, hour (24h), minute,
    second, optional fractional seconds, and optional UTC time indicator).
    This is almost equivalent to the [RFC3339](https://tools.ietf.org/html/rfc3339) 
    "date-time" format, with the exception that UTC indicator `Z` is optional and
    non-zero UTC offsets are not indicated.
    If `Z` is not indicated, the time zone is always assumed to be the local time of
    the dataset viewer.

Reply:
that looks like a nice feature to me :+1: